.gitignore

```diff
@@ -29,6 +29,10 @@
 doc/man1/openssl-*.pod
 
 # Auto generated der files
+providers/common/der/der_dsa.c
+providers/common/der/der_ec.c
+providers/common/der/der_rsa.c
+providers/common/der/der_digests.c
```

Comment:
They shouldn't exist at all at this point, as they got renamed. Should .gitignore contain historical records

Reply:
Hmm. No. My mistake - I saw these files showing in git and assumed they had been missed from .gitignore. They must have been hanging around from a build on a branch without your renaming updates.